### PR TITLE
Fix #10684: JPA case sensitive use Faces Locale

### PR DESCRIPTION
**Pull request by melloware**
Fix #10684: JPA case sensitive use Faces Locale

### CODE REVIEW

**Comment by melloware** [comment]
Actually one more fix

File: primefaces/src/main/java/org/primefaces/model/JpaLazyDataModel.java [hunk 1]
```diff
@@ -220,22 +221,24 @@ protected Predicate createPredicate(FilterMeta filter, Field filterField,
         Lazy<Collection<Object>> filterValueAsCollection = new Lazy(
                 () -> filterValue.getClass().isArray() ? Arrays.asList((Object[]) filterValue)
                         : (Collection<Object>) filterValue);
+        FacesContext context = FacesContext.getCurrentInstance();
```
Comment (tandraschko) [inline primefaces/src/main/java/org/primefaces/model/JpaLazyDataModel.java]: createPredicate is also called in a loop

Comment (melloware) [reply primefaces/src/main/java/org/primefaces/model/JpaLazyDataModel.java]: Moved it up higher outside the loops.

**Comment by tandraschko** [comment]
+1

**Pull request merged**